sklearn/model_selection/_validation.py

```diff
@@ -106,19 +114,49 @@ def cross_val_score(estimator, X, y=None, groups=None, scoring=None, cv=None,
 
     Returns
     -------
-    scores : array of float, shape=(len(list(cv)),)
+    scores : array of float or dict of such arrays, array shape=(n_splits,)
```

Comment:
But there is no way to ask for training scores, right?
So what happened to the idea of adding another function? Or do we also always add the training scores and timing etc once we return a dict?